recipe/0001-Fix-multiprocessing-test.patch

```diff
@@ -0,0 +1,17 @@
+--- rpy/tests/rinterface/test_embedded_r.py
++++ rpy/tests/rinterface/test_embedded_r.py
+@@ -74,8 +74,10 @@
+ 
++def _init_r():
++    from rpy2 import rinterface
++    rinterface.initr()
++
++
+ def test_initr():
+-    def init_r():
+-        from rpy2 import rinterface
+-        rinterface.initr()
+     preserve_hash = True
+-    proc = multiprocessing.Process(target=init_r,
```

Comment:
`osx` test failed with
```
    def dump(obj, file, protocol=None):
        '''Replacement for pickle.dump() using ForkingPickler.'''
>       ForkingPickler(file, protocol).dump(obj)
E       AttributeError: Can't pickle local object 'test_initr.<locals>.init_r'
```
Probably related to https://github.com/python/cpython/pull/13603.

TODO: upstream this patch.